README.md

```diff
@@ -34,7 +34,7 @@ The IBM Cloud MySDK Go SDK allows developers to programmatically interact with t
 
 Service Name | Package name 
 --- | --- 
-[DPX Service](https://cloud.ibm.com/apidocs) | dataproductexchangeapiservicev1
+[Data Product Exchange Services](https://test.cloud.ibm.com/apidocs/dataproducts) | dataproductexchangeapiservicev1
```

Comment:
```suggestion
[Data Product Exchange](https://cloud.ibm.com/apidocs/dataproducts) | dataproductexchangeapiservicev1
```
1. This line refers to a single service within the SDK (in general SDKs contain multiple distinct services).
2. The doc link should reflect the production version of the docs, so please remove "test." before merging.